outgoing.go

```diff
@@ -54,21 +53,20 @@ func bounce(w http.ResponseWriter, sig, url string) {
 		return
 	}
 
-	safeUrl := html.EscapeString(url)
-	if safeUrl[:10] == "javascript" {
+	if url[:10] == "javascript" {
 		debug("URL starts with javascript url: %v", url)
 		errorResp(w)
 		return
 	}
-	if !strings.Contains(safeUrl, "://") {
-		safeUrl = "http://" + safeUrl
+	if !strings.Contains(url, "://") {
```

Comment:
This isn't related to this change but if a url doesn't have a http/https protocol could it just be the error case?